### PR TITLE
Add support to return amphtml

### PR DESCRIPTION
**Pull request by raxityo**
- Looks for `<link rel="amphtml" href="https://juicy.amp.version">` and returns as part of result if found.
- Result does not include `amphtml` otherwise.

### CODE REVIEW

**Comment by alif** [comment]
CodeClimate failed. You might want to add the codeclimate config to the repo and ignore the particular check that failed since it's pretty silly.

**Comment by raxityo** [comment]
I kinda like that suggestion. ;)
Also added support for `canonical` to get canonical URL.
This will be very helpful in future for our own canonizer. Some sites have helpful query parameters that are not tracking parameters, so just stripping down query string from URL is not helpful for those URLs. By default, wordpress supports `<link rel=canonical href="*">`, so most of the publishers will have that.
Ex: Full URL `https://my-awesomeness.com?page=30&utm_source=twitter` would have canonical URL `https://my-awesomeness.com?page=30` :)

File: index.js [hunk 1]
```diff
@@ -215,16 +217,23 @@ function gatherMetaData(url, body) {
...
+      return foundValue
+    }
 
     return {
       title: (title && title.trim()) || 'Untitled',
       image: image,
-      description: description
+      description: getSimpleValue('meta[property="og:description"]', 'content'),
+      amphtml: getSimpleValue('link[rel="amphtml"]', 'href'),
+      canonical: getSimpleValue('link[rel="canonical"]', 'href')
```
Comment (alif) [inline index.js]: Interesting. Didn't know about this. Is it in the `meta` property or just in the page somewhere?

Comment (raxityo) [inline index.js]: It's found in head.

An interesting article from 2009 when it was introduced: https://webmasters.googleblog.com/2009/02/specify-your-canonical.html

Comment (alif) [inline index.js]: worth including `head` in the selector?

Comment (raxityo) [inline index.js]: We are in 'head' actually. [L193](https://github.com/mix/schenkerian/pull/48/files/753ffba569c045cc8b5d21255be9f821f2dfd998#diff-168726dbe96b3ce427e7fedce31bb0bcR193)

**Pull request merged**